native/src/components/StaticServerProvider.tsx

```diff
@@ -18,17 +18,18 @@ const staticServer = new StaticServer(SERVER_PORT, SERVER_PATH, {
 export const StaticServerContext = createContext('')
 
 const StaticServerProvider = ({ children }: StaticServerProps): ReactElement | null => {
-  const [resourceCacheUrl, setResourceCacheUrl] = useState<string | null>(null)
+  const [resourceCacheUrl, setResourceCacheUrl] = useState<string>('')
```

Comment:
Why these changes?

Reply:
Ahh. These were a remainder of when i first tried update the static server package. This was also the reason why offline capability for images did not work. Reverting this fixed that.